homeassistant/components/ness_alarm/__init__.py

```diff
@@ -63,6 +64,7 @@
             {
                 vol.Required(CONF_HOST): cv.string,
                 vol.Required(CONF_DEVICE_PORT): cv.port,
+                vol.Optional(CONF_PANEL_NAME, default="Alarm Panel"): cv.string,
```

Comment:
Same comment as in https://github.com/home-assistant/core/pull/143220#pullrequestreview-2827358846

This addition is not correct. The names for devices and entities shouldn't be set through the configuration flow.

If one wants to change the name of an entity, one can do so, through the entity settings dialog and adjust the name to their liking.

If we'd merge this proposal, one would end up with multiple places to override a name, making it unclear which one wins.

Nevertheless, thanks for being willing to contribute 👍

../Frenck